### ISSUE TITLE
Metadata not accessible

### ISSUE THREAD

**Issue by seanlinsley** [arsduo/koala#475]
Since Facebook API 2.x now just sends friends that have also used your app, it's important to instead record their total number of friends, which is sent as metadata under the "summary" key. Unfortunately, that doesn't seem to be accessible via Koala.

```ruby
>> Koala::Facebook::API.new('a token').get_connections 'me', 'friends'
=> [{"name"=>"Foo Bar", "id"=>"baz"}]
```

![screen shot 2015-06-28 at 5 45 49 pm](https://cloud.githubusercontent.com/assets/688886/8398678/24c71f78-1dbe-11e5-9214-b008af86efda.png)

**Comment by seanlinsley** [comment]
Looks like I can call `raw_response['summary']`, but ideally there should be a built-in method to access metadata.

**Comment by pratik60** [comment]
The summary field seems to be evading me as well....What do you mean by raw_response["summary"]?

**Comment by pratik60** [comment]
Thanks - Got it

```
data = Koala::Facebook::API.new('a token').get_connections 'me', 'friends'
=> [{"name"=>"Foo Bar", "id"=>"baz"}]
data.raw_response
=>  {"data" => [{"name"=>"Foo Bar", "id"=>"baz"}], "summary"=>{"total_count"=>1300}}
```

**Comment by arsduo** [comment]
Good issue! The data get returned as a GraphCollection object, which is what exposes the raw_response value. I agree that it would be useful to expose the summary value -- would either of you be interested in writing a PR for that? It'd be a simple matter of adding an attr_reader to lib/koala/api/graph_collection.rb and having that parse out the summary in the initializer.

**Comment by arsduo** [comment]
Just merged in #478 from @vhoof (thanks!) that provides the summary object directly on GraphCollection. 

@seanlinsley -- I'm releasing this now as Koala 2.2rc2 -- can you test it out and see if it meets your needs?
